instrumentation/aws_sdk/Appraisals

```diff
@@ -4,58 +4,27 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-appraise 'aws-sdk-3.1' do
-  gem 'aws-sdk', '~> 3.1'
+appraise 'aws-sdk-3' do
+  gem 'aws-sdk-core', '~> 3'
+  gem 'aws-sdk-lambda', '~> 1'
+  gem 'aws-sdk-dynamodb', '~> 1'
+  gem 'aws-sdk-sns', '~> 1'
+  gem 'aws-sdk-sqs', '~> 1'
 end
 
-appraise 'aws-sdk-3.0' do
-  gem 'aws-sdk', '~> 3.0'
+# pre-Observability support in V3 SDK
+appraise 'aws-sdk-3.2' do
```

Comment:
sdk versioning in v3 is a little weird with the aws-sdk, aws-sdk-core and aws-sdk-<servivce> gems versioned separately... 3.2 probably doesn't really make sense.  Maybe we use the core version, so 3.202?